cryptographic_estimators/MQEstimator/mq_estimator.py

```diff
@@ -144,15 +146,14 @@ def table(self, show_quantum_complexity=0, show_tilde_o_time=0, show_all_paramet
             +------------------+------+--------+-----------------------------------+------+--------+---------------------------------------------------+
             | Bjorklund        | 80.3 |   31.5 |         {'lambda_': 13/41}        | 32.9 |   32.9 |                {'lambda_': 0.19677}               |
             | BooleanSolveFXL  | 42.6 |    5.6 | {'k': 41, 'variant': 'las_vegas'} | 41.0 |    0.0 |         {'k': 41, 'variant': 'las_vegas'}         |
-            | Crossbred        | 39.8 |   37.9 |     {'D': 6, 'd': 1, 'k': 15}     | 38.0 |   37.9 |             {'D': 6, 'd': 1, 'k': 15}             |
+            | Crossbred        | 39.4 |   37.9 |     {'D': 6, 'd': 1, 'k': 15}     | 38.0 |   37.9 |             {'D': 6, 'd': 1, 'k': 15}             |
             | DinurFirst       | 57.7 |   37.9 | {'kappa': 13/40, 'lambda_': 7/40} | 28.5 |   28.5 | {'kappa': 0.3057, 'lambda_': 0.18665241123894338} |
             | DinurSecond      | 42.8 |   33.6 |             {'n1': 7}             | 33.4 |   25.8 |             {'n1': 7.592592592592592}             |
             | ExhaustiveSearch | 44.4 |   16.1 |                 {}                | 41.0 |    0.0 |                         {}                        |
-            | F5               | 57.5 |   57.0 |                 {}                | 57.5 |   57.0 |                         {}                        |
-            | HybridF5         | 43.2 |    5.4 |             {'k': 40}             | 43.2 |    5.4 |                     {'k': 40}                     |
+            | F5               | 62.9 |   57.0 |                 {}                | 57.5 |   57.0 |                         {}                        |
+            | HybridF5         | 48.6 |    5.4 |             {'k': 40}             | 43.2 |    5.4 |                     {'k': 40}                     |
```

Comment:
just to make sure: all these changes are compliant with the (corrected version of the) MQ Estimator?